client/src/utils/API.js

```diff
@@ -10,8 +13,8 @@ export default {
 
     },
     userInput: (userInput) => {
-        return axios.post("/api/search/search", userInput);
```

Comment:
we need to keep this one instead of the other one

Reply:
can you update and push again?

we need this line: return axios.post("/api/search/search", userInput);